### PR TITLE
Add prawn-dev

### PR DESCRIPTION
**Pull request by petergoldstein**
- Updates Rubocop to use prawn-dev
- Sets minimum Ruby version to 2.6
- Removes incompatible Rubies from CI
- Fixes a few lints

### CODE REVIEW

File: pdf-inspector.gemspec [hunk 1]
```diff
@@ -13,23 +13,27 @@ Gem::Specification.new do |spec|
...
+  spec.required_ruby_version = '>= 2.6'
 
   spec.add_dependency('pdf-reader', '>=1.0', '< 3.0.a')
 
   spec.add_development_dependency('bundler')
+  spec.add_development_dependency('prawn-dev', '~> 0.3.0')
   spec.add_development_dependency('rake')
   spec.add_development_dependency('rspec')
-  spec.add_development_dependency('rubocop', '~> 0.46')
   spec.add_development_dependency('yard')
```
Comment (pointlessone) [inline pdf-inspector.gemspec:37]: I guess we can drop these deps too as prawn-dev will pull them in.

File: spec/pdf_inspector_text_spec.rb [hunk 2]
```diff
@@ -3,10 +3,10 @@
 require_relative 'spec_helper'
 
 describe 'PDF::Inspector::Text' do
-  context '.analyze_file' do
+  describe '.analyze_file' do
     let(:contents) do
       PDF::Inspector::Text.analyze_file(
-        File.dirname(__FILE__) + '/fixtures/text.pdf'
+        "#{File.dirname(__FILE__)}/fixtures/text.pdf"
```
Comment (pointlessone) [inline spec/pdf_inspector_text_spec.rb:9]: Since we're changing this we might as well change it to `File.join`.

File: pdf-inspector.gemspec (hunk 1 above)
Comment (petergoldstein) [reply pdf-inspector.gemspec]: Done.

File: spec/pdf_inspector_text_spec.rb (hunk 2 above)
Comment (petergoldstein) [reply spec/pdf_inspector_text_spec.rb]: Done.

**Pull request merged**